guides/01_Getting_Started.md

```diff
@@ -0,0 +1,105 @@
+### Getting Started
+
+This guide explains how to create a meeting, add an attendee to that meeting, and use the returned information to join a meeting application to the meeting and send and receive audio. This guide assumes you have:
```

Comment:
is this correct "and use the returned information to join a meeting application **to the meeting**" ?

Reply:
Yes it is correct, though could be worded better. The meeting application is joining (as a client) of the meeting.